sale_ux/views/res_config_settings_views.xml

```diff
@@ -1,118 +1,72 @@
 <odoo>
-    <record id="res_config_settings_view_form" model="ir.ui.view">
-        <field name="name">res.config.settings</field>
-        <field name="model">res.config.settings</field>
-        <field name="inherit_id" ref="account.res_config_settings_view_form"/>
-        <field name="arch" type="xml">
-            <xpath expr="//div[@id='analytic']" position="inside">
-                <div class="col-12 col-lg-6 o_setting_box">
-                    <div class="o_setting_left_pane">
-                        <field name="analytic_account_without_company"/>
-                    </div>
-                    <div class="o_setting_right_pane">
-                        <label for="analytic_account_without_company"/>
-                        <div class="text-muted">
-                            Al confirmar ventas, crear cuenta analitica sin compañía
-                        </div>
-                    </div>
-                </div>
-            </xpath>
-        </field>
-    </record>
-    
     <record id="res_config_settings_view_form_inherit" model="ir.ui.view">
         <field name="name">res.config.settings.form.inherit</field>
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="sale.res_config_settings_view_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//div[@data-key='sale_management']/div" position="inside">
-                <div class="col-12 col-lg-6 o_setting_box">
-                    <div class="o_setting_left_pane">
-                        <field name="show_product_image_on_report"/>
+
+            <xpath expr="//div[@id='order_default_values_div']" position="after">
+                <div class="mt16">
+                    <div>
+                        <field name="group_allow_any_user_as_salesman"/>
+                        <label for="group_allow_any_user_as_salesman"/>
                     </div>
-                    <div class="o_setting_right_pane">
-                        <label for="show_product_image_on_report" string="Show Product Image On Report"/>
-                        <div class="text-muted">
-                            Show Product Image On Report
-                        </div>
+                    <div class="text-muted">
+                        By default odoo only allows to choose users wich have sales pemission in sales orders and in the sales teams only internal users. This option make able to choose, in both places, users of any type (internal or portal)
```

Comment:
By default, Odoo only allows to choose users who have sales permission in sales orders and only internal users in the sales' teams . This option enables to choose, in both places, users of any type (internal or portal)